app/design/adminhtml/default/default/template/notification/formkey.phtml

```diff
@@ -20,9 +20,9 @@
 <?php if ($this->canShow()): ?>
     <div class="notification-global notification-global-warning">
         <strong style="color:red">Important: </strong>
-        <span>Formkey validation on checkout disabled. This may expose security risks.
-        We strongly recommend to Enable Form Key Validation On Checkout in
-        <a href="<?php echo $this->getSecurityAdminUrl(); ?>">Admin / Security</a>,
-        for protect your own checkout process. </span>
+        <span>Formkey validation is disabled. This may expose you to security risks.
+        We strongly recommend to enable in
+        <a href="<?php echo $this->getEnableCSRFUrl(); ?>">Advanced / System / CSRF Protection</a>,
+        to protect your checkout, contact and newsletter forms.</span>
     </div>
```

Comment:
Do you think that this warning should be translated into the language of the chosen interface in the Backend? I think so, it should exist in a local CSV file.

Reply:
in theory it should, but:
- it was never translated before (it exists since years)
- it is very long and with dynamic content which makes it a bit harder to put in the CSV
I'd leave it like that...